src/components/card/CardList.jsx

```diff
@@ -0,0 +1,139 @@
+import './CardList.scss';
+import classNames from 'classnames';
+import testImage from 'assets/images/image_people.png';
+
+function Profile({ url }) {
+  return <img src={url} alt='profile' />;
+}
+
+/* ** ProfileList 컴포넌트 (API 사용할때 주석 부분 사용하면 됨)
+ * count = 메시지 개수
+ * messages = 받은 메시지들이 들어있는 배열
+ * first, second, third = 앞에 있는 메시지들 첫번째, 두번째, 세번째
+ * 개수에 맞게 출력하도록 했음
+ */
+function ProfileList({ count = 4, messages }) {
+  //   const recentMessages = messages.slice(0, 3);
+  //   const [first, second, third] = [...recentMessages];
+  //   const COUNT = count - 3;
+  //   return (
+  //     <>
+  //       <div className='profile-list'>
+  //         {recentMessages.map((message) => message && <Profile url={message.profileImageURL} />)}
+  //         {count > 3 && (
```

Comment:
상수 선언해주신거 이렇게 가져다 쓰면 좋을것 같습니다
```suggestion
  //         {COUNT > 0 && {
```